### ISSUE TITLE
My font is set only the first time i run the app

### ISSUE THREAD

**Issue by candeiasdan** [chrisjenx/Calligraphy#250]
If the app runs for the first time (after install) i can see my custom font. If now I stop the app and open it again the font is not being set.

on my BaseApplication class i have __onCreate()__:

```
    CalligraphyConfig.initDefault(new CalligraphyConfig.Builder()
                            .setDefaultFontPath("fonts/CustomFont.ttf")
                            .setFontAttrId(R.attr.fontPath)
                            .build());
```

MainActivity:
```
    @Override
     protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase));
     }
```

I have 3 different fonts inside my app. Whenever i need to use some specific font i do:
```
<TextView
        android:id="@+id/txtview"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:text="A"
        android:textAllCaps="true"
        android:textColor="@color/black"
        fontPath="fonts/SourceSansPro-Light.ttf"
        />
```

Am i missing something? Debugging the app, every time i open the app it passes through _onCreate()_ (BaseApplication class), _attachBaseContext_. What can be wrong here?

**Comment by chrisjenx** [comment]
What version of build tools etc are you using?
Version of Android? Version of Calligraphy? etc..

**Comment by candeiasdan** [comment]
Android Studio 2.0 Preview 5
Gradle: com.android.tools.build:gradle:2.0.0-alpha5
Library version: uk.co.chrisjenx:calligraphy:2.1.0
BuildToolVersion: 23.0.2
CompileSdkVersion: 23
MinSdkVersion: 17
Pro guard: minifyEnabled false

**Comment by chrisjenx** [comment]
Using instant run?

**Comment by candeiasdan** [comment]
No. Running the app for the first time everything works fine. Once i Stop it and re-open it, the fonts are gone.

**Comment by chrisjenx** [comment]
OK. Same issue with build tools 1.5 and release builds?

**Comment by candeiasdan** [comment]
Build tools:    classpath 'com.android.tools.build:gradle:1.5.0'
Build Variante: Release and debug.

Same problem.

**Comment by chrisjenx** [comment]
OK, Are you able to reproduce the issue in the sample app? I'm wondering if it's either the font or another library? Also are you testing on Android 5.0?

**Comment by candeiasdan** [comment]
Everything works fine in a sample app, starting and stopping, no problem.
Testing on 4.4.4

As sample app made a ListView with two row types:
1) textview where i explicitly define font family
2) textview without font family
Result: the calligraphy is applied to both layouts.

**Comment by chrisjenx** [comment]
@candeiasdan if it works in a sample app, this suggests something else is interfering. What else are you running in your app?
